Library/Homebrew/cmd/update-bash.sh

```diff
@@ -289,10 +301,16 @@ EOS
   # this procedure will be removed in the future if it seems unnecessary
   rename_taps_dir_if_necessary
 
+  DIRS=("$HOMEBREW_LIBRARY"/Taps/*/*)
+  if [[ -z "$SKIP_UPDATE_HOMEBREW_REPOSITOR" ]]
+  then
+    DIRS=("$HOMEBREW_REPOSITORY" "${DIRS[@]}")
+  fi
```

Comment:
More predictable behavior (no surprising expansions when using an unquoted variable, proper handling of white space in paths) at the cost of some Bashisms … that's the benefit, IMO.

There are far too many command line programs that fail badly because of naive assumptions about what a valid path is. I don't think we need to contribute to that, although we are restricted in what we can support ourselves because we rely on so many tools that weren't written as carefully.